internal/install/_static/terraform_deployer_run.sh

```diff
@@ -16,6 +16,15 @@ cleanup() {
 }
 trap cleanup EXIT INT TERM
 
+# Save GCP credentials on disk and perform authentication
+# NOTE: this is required for bq (and maybe other gcloud related tools) to authenticate
+export "GOOGLE_APPLICATION_CREDENTIALS=/root/.config/gcloud/application_default_credentials.json"
+echo "$GOOGLE_CREDENTIALS" > "$GOOGLE_APPLICATION_CREDENTIALS"
+gcloud auth login --cred-file "$GOOGLE_APPLICATION_CREDENTIALS"
+# NOTE: Terraform support authentication through GOOGLE_CREDENTIALS and usual gcloud ADC but other
+# tools (like bq) don't support the first, so we always rely on gcloud ADC.
```

Comment:
Could you please point me to the integration/system test, where you intend to use `bq`? I'm wondering if something changed since that time and maybe we're fine to use a [terraform module](https://github.com/terraform-google-modules/terraform-google-bigquery).

Reply:
Is being used in this PR https://github.com/elastic/integrations/pull/2312

Reply:
>This approach has multiple disadvantages: it requires more infrastructure to be created which increase complexity

What do you mean by "more infrastructure"? It sounds easier to load data on S3, than refactoring service deployer :) 

> it prevent use of emulator and there is no support to waiting for the load job completion.

This is really unfortunate and without a proper waiter it won't be possible to perform it. 

Thanks for the explainaton.

Reply:
>  It sounds easier to load data on S3, than refactoring service deployer :)

😅 totally true. 

If `google_bigquery_job` was working as expected I would have chosen that way, because at least is predictable and all within terraform, but I think we could only have something fragile at the moment.